ehmurray8/KytonUI
# Fix sensitivity

#### What this PR does?
This PR fixes the sensitivity curves to properly plot a single sensitivity curve instead of a fit curve for each cycle. This also adds the linear slope coefficients to the baking graphs, and separates the bake curves into their own plots.

#### How should this be manually tested?
This should be tested by creating an excel calibration spreadsheet and verifying the calibration sensitivity curves. Also create an excel file for a bake, and make sure the graphs are split for each FBG, and the curve fit coefficients are correct.

#### Background context you want to provide?
This was done to properly report the sensitivity to the user, and to give the user a baking trendline.

#### What are the relevant issues?
#129 #128
